### PR TITLE
Add Vietnamese privacy policy.

### PR DESCRIPTION
**Pull request by imthinhvu**
- [x] I have put a reference to a related issue in the description
- [x] I have created the pull request from the correct version branch
- [x] I have ensured the branch is rebased and/or there are no merge conflicts

Addresses #19. 

My Vietnamese grammar is not the greatest, so would appreciate some proofreading. I gave it my best shot.

### CODE REVIEW

**Comment by jyntran** [comment]
@imthinhvu Your Vietnamese is much better than mine! I will do my best to proofread.

File: privacy-vn.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Chính Sách Về Quyền Riêng Tư
+## Thông Tin Nào Được Thu Thập?
+Tiện ích NhacCuaTui Scrobbler sự dụng API Last.fm để lấy được xác thưc. Kết quả là tên người dùng và một mật khẩu đặc biệt cho sự dụng.
+
+Tiện ích chỉ chạy được trên các trang từ nhacuatui.com, va tìm kiếm tên bài hát và tên nghệ sĩ của bài đang nghe.
```
Comment (jyntran) [inline privacy-vn.md]: There is a missing `c` in `nhacuatui.com`.

Also does `va` need to be changed to `và`?

File: privacy-vn.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+## Thông Tin Sự Dụng Thế Nào?
+Mật khẩu cho phép để lượt qua các bài hát của tài khoản. Tên người dùng sẽ chiếu trên bặng tiện ích. Tiện ích sẽ thâu thông tin bài nhạc của NhacCuaTui va gủi đến Last.fm.
```
Comment (jyntran) [inline privacy-vn.md]: Same here, `va` to `và`?

Comment (imthinhvu) [reply privacy-vn.md]: Thanks, made the changes!

**Comment by jyntran** [comment]
@imthinhvu Looks good. Thanks for contributing!

**Pull request merged**